game_eggs/steamcmd_servers/lastoasis/egg-last-oasis.json

```diff
@@ -0,0 +1,125 @@
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:6",
+            "field_type": "text"
+        },
+        {
+            "name": "Server Slots",
+            "description": "Limited to 100",
+            "env_variable": "SLOTS",
+            "default_value": "5",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:100",
+            "field_type": "text"
+        },
+        {
+            "name": "Provider Key",
+            "description": "You will need to login to \r\n\r\nhttps:\/\/myrealm.lastoasis.gg\/ \r\n\r\nusing Steam, and grab a few keys used for the server setup. Specifically, you will require a CustomerKey, which is used in all servers you\u2019re using on that realm, and a ProviderKey.\r\n\r\nProviderKey can be utilized as many times as you want in as many different servers in the same realm. CustomerKey must be the same across all servers in a realm.",
+            "env_variable": "PROVIDER_KEY",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:50",
+            "field_type": "text"
+        },
+        {
+            "name": "Customer Key",
+            "description": "You will need to login to \r\n\r\nhttps:\/\/myrealm.lastoasis.gg\/ \r\n\r\nusing Steam, and grab a few keys used for the server setup. Specifically, you will require a CustomerKey, which is used in all servers you\u2019re using on that realm, and a ProviderKey.\r\n\r\nProviderKey can be utilized as many times as you want in as many different servers in the same realm. CustomerKey must be the same across all servers in a realm.",
+            "env_variable": "CUSTOMER_KEY",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:20",
+            "field_type": "text"
+        },
+        {
+            "name": "LD Lib Path",
+            "description": "",
+            "env_variable": "LD_LIBRARY_PATH",
+            "default_value": ".\/linux64",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|max:20",
+            "field_type": "text"
+        },
+        {
+            "name": "Auto Update",
+            "description": "",
+            "env_variable": "AUTO_UPDATE",
+            "default_value": "1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|boolean",
+            "field_type": "text"
+        },
+        {
+            "name": "[Advanced] Branch Name",
+            "description": "",
+            "env_variable": "SRCDS_BETAID",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "string|nullable|in:public,classic",
```

Comment:
you eather set the default to public or just remove the validation or do `string|in: ,classic`